subscriptions/views/widget_library.py

```diff
@@ -0,0 +1,25 @@
+from django.http import JsonResponse
+from rest_framework.decorators import api_view
+
+from associations.models import Library
+
+
+@api_view(["GET"])
+def widget_library_view(request, library_id):
+    """Displays the most borrowed loanables if they are available"""
+
+    marketplace = Library.objects.get(pk=library_id)
+
+    response = {}
+
+    suggested_loanables = marketplace.loanables.filter(
+        real_return_date__is_null=False
```

Comment:
Does `real_return_date` exist in `Loanable`? I can find it in `Loan`, but not in `Loanable` (on the branch I am currently in)